BusBoard.ConsoleApp/Program.cs

```diff
@@ -15,16 +17,34 @@ class Program
     static void Main(string[] args)
     {
       ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
-      var clientResponse = getClientJSONResponse();
+
+      var clientJSONResponse = getClientJSONResponse();
+      List<BusData> listArrivingBuses = getArrivingBussesFromJSON(clientJSONResponse);
+      printBusesToConsole(listArrivingBuses);
+    }
+
+    private static void printBusesToConsole(List<BusData> listArrivingBuses)
+    {
+      Console.WriteLine($"The next 5 buses arriving at {listArrivingBuses[0].StationName} are: \n");
+      foreach (var bus in listArrivingBuses)
+      {
+        Console.WriteLine(bus.VehicleID);
+      }
+    }
+
+
+    private static List<BusData> getArrivingBussesFromJSON(IRestResponse clientJSONResponse)
+    {
+      List<BusData> listArrivingBusDataFromJSON = JsonConvert.DeserializeObject<List<BusData>>(clientJSONResponse.Content);
+
+      return listArrivingBusDataFromJSON;
     }
 
     private static IRestResponse getClientJSONResponse()
```

Comment:
I think you could move all of the TfL API logic out in to it's own class to encapsulate it and separate it from the main method and program class.